src/styles/card.scss

```diff
@@ -0,0 +1,75 @@
+.grid-container {
+  display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(200px, 200px));
+  justify-content: center;
+  gap: 46px 48px;
+  padding: 50px 40px;
+}
+
+.card {
+  display: flex;
+  justify-content: center;
+  flex-direction: column;
+  width: 200px;
+  border: $gray-border;
+  padding: 32px 16px 16px;
+  border-radius: $rounding;
+  transition: $transition-duration;
```

Comment:
Ensure that the SCSS variable `$transition-duration` is defined in your stylesheets. If it's not defined, it will cause an error during the SCSS compilation.